### DISCUSSION TITLE
High error rate at DL

### DISCUSSION THREAD

**Discussion by pengnist** [srsran/srsRAN_Project#213] [General Help] [answered]
Hi Community,

We are having a few setups running with srsRAN gNB, Open5GS, FlexRIC, and srsue running 1) in separate servers, and 2) with srsRAN gNB, Open5GS, and FlexRIC in the same server. We have tried the connection between gNB and srsue with 1) zmq and 2) USRPs with RF cables and attenuators. In all these deployments, we experience high DL error rate. And after some time, the DL error rate may reach 100%, and then stopped transmission, while the UL still shows traffic, even if application (ping or iperf3) already stopped. I am wondering if anyone has good suggestions how to start the troubleshooting. Could it be the configuration of the system or software?

The servers for the components except srsue have Intel(R) Xeon(R) Gold 6334 CPU @ 3.60GHz. 

I attach some screenshots with ZMQ connection and ping 8.8.8.8 -I tun_srsue:

![image](https://github.com/srsran/srsRAN_Project/assets/71417513/30329e60-1de2-445a-9bec-fa5a22fa9575)

DL stopped:

![image](https://github.com/srsran/srsRAN_Project/assets/71417513/02228d22-9d64-4cdc-a1b8-a9d42c05dde5)

Thanks

**Comment by andrepuschmann** [comment]
It's a bit hard to help since you're not providing any logs. Maybe start with one scenario, upload logs and we have a look. But with ZMQ at least you're not supposed to see any BLER of course.

**Reply by pengnist** [reply]
Thanks for your quick response. Are there any specific logs you are interested in? Like gnb.log? I will have to edit the IP to hide institute related info.

**Reply by pengnist** [reply]
Please find the attached gnb.log (it has to be compressed to meet size limit). The setup is as follows:
![image](https://github.com/srsran/srsRAN_Project/assets/71417513/c72060a9-2b29-4f18-95b2-b76f4aaf69f9)
[gnb.zip](https://github.com/srsran/srsRAN_Project/files/12650082/gnb.zip)

**Reply by pengnist** [reply]
Here is the ue.log:
[ue.zip](https://github.com/srsran/srsRAN_Project/files/12651492/ue.zip)

**Reply by pengnist** [reply]
I am not sure if this info will be useful to find out the cause of the issue, but we are currently using low latency kernel.

**Reply by andrepuschmann** [reply, marked as answer]
Could you please try again first with the current `main` release of the gnb and leave all RIC things disabled.

**Reply by andrepuschmann** [reply]
Is the issue gone?

**Reply by pengnist** [reply]
Yes, thanks!
The only issue with the new update is that the E2 binder address needs to be specified, if RIC is not in the same machine as the gnb
